templates/github/.ci/ansible/Containerfile.j2.copy

```diff
@@ -7,6 +7,12 @@ ADD {{ item.source }} {{ item.source }}
 {% endif %}
 {% endfor %}
 
+{%- for item in plugins -%}
```

Comment:
@dkliban it seemed like (and I might be wrong about this) that the pip dependencies are installed each time CI is run. Would it be possible to install the basic pulpcore requirements [here](https://github.com/pulp/pulp-oci-images/blob/latest/pulp_ci_centos/Containerfile), and then run `pip install --upgrade` when the plugins are installed in CI?

A large chuck of the CI time is just downloading the pip packages during the container build. Would be nice to make that run faster.

Reply:
That is a good idea.  I would like to implement it as a separate change from this.